cluster/gce/config-default.sh

```diff
@@ -123,6 +123,12 @@ ENABLE_L7_LOADBALANCING="${KUBE_ENABLE_L7_LOADBALANCING:-glbc}"
 #   standalone     - Heapster only. Metrics available via Heapster REST API.
 ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-influxdb}"
 
+# Optional: Enable Metrics Server. Metrics Server should be enable everywhere,
+# since it's a critical component, but in the first release we need a way to disable
+# this in case of stability issues.
+# TODO(piosz) remove this option once Metrics Server became a stable thing.
+ENABLE_METRICS_SERVER="${KUBE_ENABLE_METRICS_SERVER:-true}"
+
```

Comment:
If this is going to be enabled by default, we should be using it for something, and should not deploy (or at least warn about) Heapster.  Turn it on in a separate PR once we're ready to make the HPA use it by default.

Reply:
I'd like  to switch HPA in 1.8 to use metrics server.

Heapster is used by dashboard. Also we should perform a graceful switch and have both metrics-server and Heapster enabled in one release (1.8).

Reply:
Ah, ack.